instrumentation/grpc/README.md

```diff
@@ -60,6 +60,28 @@ overrideSpanName = new GrpcClientParser() {
 };
 ```
 
+## gRPC Propagation Format (Census interop)
+
+gRPC defines a [binary encoded propagation format](https://github.com/census-instrumentation/opencensus-specs/blob/master/encodings/BinaryEncoding.md) which is implemented
+by [OpenCensus](https://opencensus.io/) instrumentation. When this is
+the case, incoming requests will have two metadata keys "grpc-trace-bin"
+and "grpc-tags-bin".
+
+When enabled, this component can extract trace contexts from these
+metadata and also write the same keys on outgoing calls. This allows
+transparent interop when both census and brave report data to the same
+tracing system.
+
+To enable this feature, set `grpcPropagationFormatEnabled` which is off
+by default:
+```java
```

Comment:
To give most flexibility I made this a feature flag similar to how tracing is still a flag in gRPC